features/steps/laravel_steps.rb

```diff
@@ -3,4 +3,25 @@
     When I set environment variable "BUGSNAG_CAPTURE_SESSIONS" to "true"
     And I set environment variable "BUGSNAG_SESSION_ENDPOINT" to "http://#{current_ip}:#{MOCK_API_PORT}"
   }
+end
+
+When(/^I start the laravel fixture$/) do
+  laravel_fixture = ENV['LARAVEL_FIXTURE'] || 'laravel56'
```

Comment:
For supporting other versions including Lumen & Laravel 5.7 this should probably be a function call to a lookup table